src/crypto/crypto_tls.h

```diff
@@ -136,7 +136,8 @@ class TLSWrap : public AsyncWrap,
           v8::Local<v8::Object> obj,
           Kind kind,
           StreamBase* stream,
-          SecureContext* sc);
+          SecureContext* sc,
+          bool stream_has_active_write);
```

Comment:
Non-blocking nit: I'm generally not a fan of bool arguments to methods, largely because just seeing the `true` or `false` a the callsite tends not to be descriptive enough to let folks know what the impact is. I certainly won't block this change because of it, but I'd prefer a enum arg here instead.